ui/eureka/eureka.js

```diff
@@ -2150,6 +2171,13 @@ class EurekaTracker {
         container.removeChild(label);
     }, this.options.FlagTimeoutMs);
   }
+
+  AddFairy(name, ex, ey) {
+    let zi = this.zoneInfo;
+    let mx = zi.entityToMapXScalar * ex + zi.entityToMapXConstant;
+    let my = zi.entityToMapYScalar * ey + zi.entityToMapYConstant;
+    this.AddFlag(mx, my, name, "");
```

Comment:
For consistency, it might be worth just making the name "fairy".  That's what other code does if folks link to a fairy in chat.

Reply:
It actually already goes through `SimplifyText` inside `AddFlag` so it already gets translated to "fairy" because of the "elemental" in its name. But yeah, might as well change it here, especially w.r.t other languages and stuff.

I think ultimately we would just mark it with its own icon and no text anyways.